packages/php-wasm/node/src/test/php.spec.ts

```diff
@@ -554,3 +558,85 @@ bar1
 		});
 	});
 });
+
+// @TODO Prevent crash on PHP versions 5.6, 7.2, 8.2
```

Comment:
Can we create a new issue to track this?

Reply:
I made a note in https://github.com/WordPress/wordpress-playground/issues/273